sample/general/jobs/sample_pytest.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+"""Pytest sample for demo."""
+import azure.ai.ml
+from azure.ai.ml import MLClient
+from azure.identity import DefaultAzureCredential
+import os
+from pathlib import Path
+
+
+def submit_pytest_job():
+    """Submit a job."""
+    subscription_id = os.environ.get('subscription_id')
+    resource_group = os.environ.get('resource_group')
+    workspace = os.environ.get('workspace')
+    ml_client = MLClient(DefaultAzureCredential(), subscription_id, resource_group, workspace)
+    parent = Path(__file__).resolve().parent
+    test_job = azure.ai.ml.load_job(parent / 'pipeline.yml')
+    print('Running test job pipeline.yml')
+    test_job = ml_client.jobs.create_or_update(test_job)
+
+    submitted_job_list = [test_job]
+
+    while submitted_job_list:
```

Comment:
If you're going to poll you should also include a delay per iteration of this loop so that you don't hammer the backend.

Reply:
done